data/stats/stats.json

```diff
@@ -0,0 +1,66 @@
+[
```

Comment:
I'm not sure what this file could be used for?

Reply:
This file (like most of the others) are mainly about formalising specific definitions with in each ship. It also provides a link between the FFG definition and the XWS definition of a ships statistics in this particular case.

EG When you look at the Tie/ln fighter the XWS contains the following statistics about a ship. The "type" is just a string that must be common across all ships and if it needs to change for some reasons the definition (eg and if we wish to add further definition we would to find and adjust every instance. By centralising them around an xws "tag" we can then store things locally. It also allows us to validate our XWS against the FFG source for each ship easily.

EG XWS ---
"stats": [
    { "arc": "Front Arc", "type": "attack", "value": 2 },
    { "type": "agility", "value": 3 },
    { "type": "hull", "value": 3 }
],

EG FFG ---
"statistics": [
                {
                    "id": 7000,
                    "statistic_id": 10,
                    "value": "2",
                    "recurring": false
                },
                {
                    "id": 7001,
                    "statistic_id": 1,
                    "value": "3",
                    "recurring": false
                },
                {
                    "id": 7002,
                    "statistic_id": 2,
                    "value": "3",
                    "recurring": false
                }
            ],

Reply:
That makes sense.

Reply:
The only comment I have then is about the naming of the arc. They're a bit different in xwing-data